THEMING.md

```diff
@@ -0,0 +1,134 @@
+<!--docs:
+title: "Theming"
+layout: detail
+section: components
+excerpt: "Material Theming refers to the customization of your Material Design app to better reflect your product’s brand."
+-->
```

Comment:
https://github.com/material-components/material-components-ios/tree/develop/components/schemes is the current location of our theming docs. Are we able to update that document instead?

Reply:
Done

Reply:
Can we remove this file from the PR as well now?

Reply:
Whoops I thought I deleted it last commit.